bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -1478,17 +1478,18 @@ namespace JsRTApiTest
     {
         JsValueRef result = JS_INVALID_REFERENCE, callbackState = JS_INVALID_REFERENCE, exception = JS_INVALID_REFERENCE;
         JsValueType cbStateType = JsUndefined;
+        const wchar_t *script = _u("new Promise((res, rej) => res()).then(() => 1)");
```

Comment:
I think you want to use WCHAR since wchar_t is not a consistent size across Linux and Windows.